src/components/EditCoursePage/index.jsx

```diff
@@ -0,0 +1,213 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+
+import EditCourseForm from './EditCourseForm';
+import StatusAlert from '../StatusAlert';
+import LoadingSpinner from '../LoadingSpinner';
+
+
+class EditCoursePage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      startedFetching: false,
+    };
+    this.handleCourseEdit = this.handleCourseEdit.bind(this);
+    this.setStartedFetching = this.setStartedFetching.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.fetchCourseInfo();
+    this.props.fetchCourseOptions();
+    this.setStartedFetching();
+  }
+
+  setStartedFetching() {
+    this.setState({ startedFetching: true });
```

Comment:
I am confused as to why we need to set state in the view?  

Could this be handled with the redux actions and reducers?

If the first thing we are doing is fetching shouldn't the initial state be fetching?

Reply:
Re question 1: we are using this state variable as an indicator for when we have actually started fetching or not. This is because isFetching is initially False and without having this state set, it will try to render the component without any data and cause it to crash.

Re question 2: It might be able to be handled with redux, but I couldn't come up with a good way to set it and thought state might be the best. 

Re question 3: I kind of disagree with that because there is a time period before we start fetching so initializing it to `true` feels inaccurate.

Reply:
It sounds like you have thought it out.  I do not feel that I need to block on this change.

Thank you for the detail.